### PR TITLE
update and layout docs

### PR DESCRIPTION
**Pull request by GeorgWa**
- Update and layout docs to be more consistent
- Fix broken Links

### CODE REVIEW

File: docs/methods/search_parameter_optimization.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+## Overview of optimization
+AlphaDIA can perform optimization for the following parameters, `target_ms1_tolerance`, `target_ms2_tolerance`, `target_mobility_tolerance` and `target_rt_tolerance`. There are two optimization strategies, targeted and automatic.
+
+### Targeted optimization
+The search space is progressively narrowed until a target tolerance is reached for a given parameter.
+
+To activate targeted optimization for example for fragment m/z tolerance, set `target_ms2_tolerance` to `10` for using a target tolerance of 10 ppm.
+For retention time, the target value can be either set as an absolute value in seconds or as a fraction of the total retention time range.
+
+For example, setting `target_rt_tolerance` to `300` will result in a target tolerance of 300 seconds, while setting it to `0.3` will use 30% of the gradient length as the target tolerance.
```
Comment (mschwoer) [inline docs/methods/search_parameter_optimization.md:16]: is `0.3` a realistic value to use?

File: docs/methods/search_parameter_optimization.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
-# Search Parameter Optimization
+# Optimization and Calibration
+In peptide centric DIA search, calibration of the library and optimization of search parameters is required to maximize the number of confident identifications. AlphaDIA performs both calibration and optimization iteratively. Calibration removes the systematic deviation of observed and library values to account for technical variation from the LC or MS instrument. Optimization reduces the search space to improve the confidence in identifications and to accelerate search.
+:::{note}
+Calibration and optimization are different but connected to transfer learning. In [transfer learning](./transfer-learning.md) the residual (non-systematic) variation is learned and thereby reduced. This usually leads to better performance if used with optimization and calibration.
```
Comment (mschwoer) [inline docs/methods/search_parameter_optimization.md:4]: (nit) `but both connected to transfer learning`

File: docs/methods/search_parameter_optimization.md [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+
+To activate targeted optimization for example for fragment m/z tolerance, set `target_ms2_tolerance` to `10` for using a target tolerance of 10 ppm.
+For retention time, the target value can be either set as an absolute value in seconds or as a fraction of the total retention time range.
+
+For example, setting `target_rt_tolerance` to `300` will result in a target tolerance of 300 seconds, while setting it to `0.3` will use 30% of the gradient length as the target tolerance.
+
+### Automatic optimization
+In automatic optimization the search space is reduced until an optimal value is detected. This optimization is curently performed for every raw file individually. The results of the optimization can be found in the [stats.tsv](<project:../methods/output-format.md>) file in the output directory.
+
+To activate automatic optimization for example for retention time, set `target_rt_tolerance` to `0.0`.
```
Comment (mschwoer) [inline docs/methods/search_parameter_optimization.md:21]: (nit) To activate automatic optimization for a certain quantity, set the respective target to `0.0`, for example set `target_rt_tolerance=0.0` for automatic retention time optimization.

File: docs/methods/search_parameter_optimization.md [hunk 4]
```diff
@@ -20,12 +44,32 @@ If enough confident target precursors have been detected, they are calibrated to
...
 
-The optimization is finished as soon as the minimum number of steps `min_steps` has passed and all tolerances have either 1) reached the target tolerances defined in `search` if using targeted optimization, or 2) have converged if using automatic optimization.
+The optimization is finished as soon as the minimum number of steps `min_steps` has passed and all tolerances have either 1. reached the target tolerances defined in `search` if using targeted optimization, or 2. have converged if using automatic optimization.
 
 ## Configuring calibration and optimization
 The configuration below will perform targeted optimization of precursor m/z, fragment m/z and ion mobility, and automatic optimization of retention time.
 
 ```yaml
+search:
+  # Number of peak groups identified in the convolution score to classify with target decoy competition
```
Comment (mschwoer) [inline docs/methods/search_parameter_optimization.md:54]: I would not repeat the general explanations here, rather move them to the `default_config.yaml` and reference it here..
I would comment here only if something is very specific to the described problem..

**Review by mschwoer — APPROVED** [review]
LGTM!

File: docs/methods/search_parameter_optimization.md (hunk 1 above)
Comment (GeorgWa) [reply docs/methods/search_parameter_optimization.md:16]: yes

**Pull request merged**